src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/personalrelationships/ContactPersonReconciliationService.kt

```diff
@@ -93,18 +94,8 @@ class ContactPersonReconciliationService(
         async { dpsApiService.getPrisonerContacts(prisonerId.offenderNo).content!! }
     }.awaitBoth()
 
-    val dpsContactSummaries = dpsContacts.map { it.asSummary() }.sortedWith(
```

Comment:
abandoned the idea of ordering and compare pairs - it doesn't work when there are duplicates like there are In live due to bad data